model/labelname.go

```diff
@@ -34,10 +34,8 @@ const (
 	// label names.
 	ReservedLabelPrefix = "__"
 
-	// HiddenLabelPrefix is a prefix which is legal in user-supplied label names
-	// but will not appear in the eventual metrics.
-	// Reserved labels may be excepted from that rule.
-	HiddenLabelPrefix = "_"
+	// MetaLabelPrefix is a prefix for labels that provide meta information.
```

Comment:
Certain yes to the final part. `Meta` is completely generic though and we might use it outside of SD in the future.